### PR TITLE
Added keyboard jkdlab/binary_monkey

### PR DESCRIPTION
**Pull request by LDobac**
Hi! We made 3-Key macropad keyboard inspired by monkey!
It keyboard consists of 1, 0 and Enter Keys.
And We also made funny keymap called Ascii keymap. If press 0 and 1, it will record by binary and press enter it represent to ascii code.

## Description

This keyboard 3-Key (1x3 matrix) macropad.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/jkdlab/binary_monkey/readme.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Binary_Monkey
+
+![Binary_Monkey](https://i.imgur.com/0meBVKT.png?1)
```
Comment (fauxpark) [inline keyboards/jkdlab/binary_monkey/readme.md:3]:
```suggestion
![Binary_Monkey](https://i.imgur.com/0meBVKTh.png)
```

File: keyboards/jkdlab/binary_monkey/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+* Keyboard Maintainer: [JKDLAB.](https://github.com/jkdlab) [yikolden](https://github.com/LDobac)
+* Hardware Supported: atmega32u4
+* Hardware Availability: [Gumroad](https://gumroad.com/jkdlab)
+
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make jkdlab/binary_monkey:default
+
```
Comment (fauxpark) [inline keyboards/jkdlab/binary_monkey/readme.md:16]:
```suggestion
Make example for this keyboard (after setting up your build environment):

    make jkdlab/binary_monkey:default
```

File: keyboards/jkdlab/binary_monkey/rules.mk [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+MOUSEKEY_ENABLE = no          # Mouse keys
+EXTRAKEY_ENABLE = yes         # Audio control and System control
+CONSOLE_ENABLE = no           # Console for debug
+COMMAND_ENABLE = no           # Commands for debug and configuration
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no              # USB Nkey Rollover
+BACKLIGHT_ENABLE = no         # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = no          # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no         # Enable Bluetooth
+AUDIO_ENABLE = no             # Audio output
```
Comment (fauxpark) [inline keyboards/jkdlab/binary_monkey/rules.mk:19]:
```suggestion
# Bootloader selection
BOOTLOADER = atmel-dfu

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/jkdlab/binary_monkey/config.h [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+ 
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID       0x0324
```
Comment (drashna) [inline keyboards/jkdlab/binary_monkey/config.h:23]: This Vender ID is already in use, and needs to be changed. 

https://www.the-sz.com/products/usbid/index.php?v=0x0324&p=&n=

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/jkdlab/binary_monkey/config.h (hunk 4 above)
Comment (LDobac) [reply keyboards/jkdlab/binary_monkey/config.h]: Hi! I changed VENDOR_ID not used. Thank you!

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]
Thanks!

**Review by drashna — APPROVED** [review]
Thanks!

**Pull request merged**